package.json

```diff
@@ -92,6 +92,7 @@
     "octokit": "^2.0.14",
     "path-to-regexp": "^6.2.1",
     "posthog-js": "^1.83.0",
+    "posthog-node": "^3.1.3",
```

Comment:
Note that we're using feature flag support server-side because some browsers and/or browser extensions can block PostHog.

![image](https://github.com/open-sauced/app/assets/833231/e6f41333-b41e-472e-b837-1d097a9ed903)